### PR TITLE
Feature branch1

### PR DESCRIPTION
**Pull request by willy836**
Create a branch called feature-branch1

### CODE REVIEW

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @willy836 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive commit messages ✔️

## Required Changes ♻️

- [ ]  Kindly update your readme by using [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md)
- [ ] Kindly set up linters correctly by following [this link](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [ ] Also consider adding code for index HTML with `h1` contain hello world
- [ ] Consider adding a descriptive pull request by clicking in edit button and adding a description with what you did in this project like: `I created my hello world project`, `I set up linters`, `I styled `h1``

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by willy836** [comment]
I updated my hello word project, i set up linters, added a descriptive readme, and added a header in the index.html file

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+HTML
+CSS
+
+Authors
+GitHub: @willy836
+Twitter: @wkimiri
+LinkedIn: Wilson Kimiri
+
+Contributing
+Contributions, issues, and feature requests are welcome!
```
Comment (vickymarz) [inline README.md]: - [ ] You have done well updating your readme 😄 as the previous reviewer suggested, but I think you should improve on your readme by following the template provided by Microverse. Please ensure your readme looks like the[ template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided 👍🏼 .

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @willy836 , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 

 ## Required Changes ♻️

- [ ] As the previous reviewer has suggested, Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up linters correctly. Currently, your linters are lacking the stylelint and the web hint config files 👍🏼 . 

- [ ] Kindly reminder to add node_modules to gitignore which will be generated while setting up linters in your project 😄.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by willy836** [comment]
hello
after setting up my linters, node_modules were not generated. help

**Comment by willy836** [comment]
When setting up my linters, I noticed the npm commands were not working, so I just copied and pasted the files. Also, no node_modules were generated. I updated the readme following the template provided.

**Comment by vickymarz** [comment]
@willy836 did you do npm init -y ?

**Review by zhadier — APPROVED** [review]
## Hi @willy836  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏
## Optional
- [ ] [ _Optional_ ] Kindly always add `node_modules` in your `.gitignore` file as a best practice, irrespective of whether they exist in the project or not. There's no harm in adding it to .gitignore even if the node_modules do not exist. It can be considered a safeguard if you decide to add dependencies and generate modules in the future.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by willy836** [comment]
@

Okoroji Victor Ebubeyes. i did npm init -y but it still didnt work

**Comment by vickymarz** [comment]
@willy836 if you run the commands in the linters.yml for the webhint and stylint installation, you will see the node modules generated. It seems you didn't run those commands in your terminal.

**Comment by willy836** [comment]
@Victor, yes. They are running on terminal. I was previously running the
commands on gitbash. Thanks!!

**Pull request merged**